DesignSafe-CI/portal
# DES-2438: Related Work and Referenced Data text updates

## Overview: ##
Text updates from feedback from DES-2115

## PR Status: ##


## Related Jira tickets: ##

* [DES-2438](https://jira.tacc.utexas.edu/browse/DES-2438)

## Summary of Changes: ##
Moved display of Related Work and Referenced Data to html snippets/directives
Updated text as requested in the jira task
Fixed a few places that got missed in the DES-1499 that still needed classes for better formatting

## Testing Steps: ##
1. Make sure every location that shows Related Work it shows the Type if it exists.
2. Make sure every location that should show Referenced Data shows it if it exists.

## UI Photos:

## Notes: ##
This might have a merge conflict with Vanessa's PR related to the citation updates, though I hope not.
Older projects with older metadata might have:
- no Related Work Type to display
- Referenced Data types which are not available anymore but might still display if it previously existed
- "wrong" Related Work Type - Linked Project is now Linked Dataset, but older metadata might still show Linked Project
